README.md

```diff
@@ -38,22 +41,24 @@ Ruffle supports ActionScript 1, 2 and 3 pretty well, but it's still not finished
 
 ## Using Ruffle
 
-The easiest way to try out Ruffle is to visit the [web demo page](https://ruffle.rs/demo/), then click the "Browse..." button to load an SWF file of your choice.
+The easiest way to try out Ruffle is to visit the [web demo page](https://ruffle.rs/demo/), then click the "Select File" button to load an SWF file of your choice.
 
-[Nightly builds](https://ruffle.rs/#releases) of Ruffle are available for desktop and web platforms including the browser extension.
+[Nightly builds](https://ruffle.rs/downloads#nightly-releases) of Ruffle are available for desktop and web platforms.
 
 For more detailed instructions, see our [wiki page](https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle).
 
 ## Building from source
 
-[Follow the official guide](https://www.rust-lang.org/tools/install) to install Rust for your platform.
+### Prerequisites
 
-You must also have Java installed, and available on your PATH as `java`.
+* Latest stable channel of [Rust](https://www.rust-lang.org/tools/install)
+* Java, available on your PATH as `java` (required for building the library containing the builtin Flash classes for ActionScript 3)
 
 ### Desktop
 
-If you are building for a Linux platform, the following are typical dependencies:
-#### Ubuntu
+#### Linux prerequisites
```

Comment:
Strictly speaking, some of these are general linux prerequisites - not specifically desktop.
I know that's not new to this PR though :D

Reply:
Might be worth it to add a note that these are Ubuntu packages and that the names might not match with other flavors of Linux (e.g. Fedora), just for example https://github.com/ruffle-rs/ruffle/pull/14654#issuecomment-2029760510.

Reply:
I'm not a Linux expert, but I know that at least [Arch](https://aur.archlinux.org/packages/ruffle-nightly-bin) depends on `alsa-lib`, `openssl`, `zlib`, and `libxcb`. Might worth adding these as well?

Reply:
> I'm not a Linux expert, but I know that at least [Arch](https://aur.archlinux.org/packages/ruffle-nightly-bin) depends on `alsa-lib`, `openssl`, `zlib`, and `libxcb`. Might worth adding these as well?

`libasound2` should be equivalent to `alsa-lib`, I think it might be worth it to have separate lists for different flavors.

> Also, we might also want to delete `default-jre-headless` from the list as Java is already present in the general prerequisites.

I think we can keep it in, no harm mentioning it twice, just in case the reader missed it the first time.